### ISSUE TITLE
False positive for dead link

### ISSUE THREAD

**Issue by biesiadamich** [sindresorhus/awesome-lint#107]
Hi,

False positives for these links:
```
Link to https://www.pexels.com/photo/bandwidth-close-up-computer-connection-1148820/ is dead
Link to https://www.pexels.com/photo/interior-of-office-building-325229/ is dead
```

Generally it works correctly.

Best wishes,

**Comment by biesiadamich** [comment]
I've got a solution in this case, links changed as:
https://images.pexels.com/photos/325229/pexels-photo-325229.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260
https://images.pexels.com/photos/1148820/pexels-photo-1148820.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260
are fine.

(~directly).

**Comment by sindresorhus** [comment]
See: https://github.com/transitive-bullshit/check-links/issues/4

Doesn't look like that will be fixed anytime soon. So I think we should just remove the rule for now. It's more annoying than useful.

**Comment by transitive-bullshit** [comment]
Agreed; this rule has failed to meet expectations and unfortunately is too flaky in practice.

**Comment by biesiadamich** [comment]
I think we can close the issue.
